tests/container-scripts/upgrade-test-internal.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Number of validators to start
+NODES=$1
+# what test to execute
+TEST_TYPE=$2
+ALCHEMY_ID=$3
+set -eux
+
+# This must sync with the --mount argument passed to docker in run-upgrade-test.sh
+OLD_BINARY_LOCATION=/oldgravity
+
+# Stop any currently running gravity and eth processes
+pkill gravity || true # allowed to fail
+pkill geth || true # allowed to fail
+
+# Wipe filesystem changes
```

Comment:
ah I see this is a helper script for running the upgrade test live. a little bit more doc on how things should be setup before running this would be nice, but if we move to the more integrated update test structure we won't need it. Your call.